app/mailers/report_mailer.rb

```diff
@@ -5,10 +5,29 @@ def budget_status
     to_email = ENV.fetch('BUDGET_REPORT_RECIPIENT')
     date = Time.now.utc.strftime("%a %m/%d/%y (%Z)")
 
+    build_attachments
+
     mail(
       to: to_email,
       subject: "C2: Daily Budget report for #{date}",
       from: self.sender_email
     )
   end
+
+  private
+
+  # rubocop:disable AbcSize, TrailingComma
```

Comment:
Feel free to disable the `TrailingComma` check in the `.rubocop.yml`, but please fix the `AbcSize` issue!